src/code/mixins/app-view.coffee

```diff
@@ -98,9 +98,10 @@ module.exports =
       else
         undo = redo = false
       if undo or redo
-        e.preventDefault()
-        @props.linkManager.redo() if redo
-        @props.linkManager.undo() if undo
+        if (@props.linkManager.undoRedoIsVisible)
+          e.preventDefault()
```

Comment:
If the user presses command-z or ctr-z, does it get handled by CODAP here? 
It would be good to notify CODAP the users intention of wanting to undo the last command if we can.